.travis.yml

```diff
@@ -8,4 +8,4 @@ before_install:
 script:
   # the "storage-ng-travis-ruby" script is included in the base yastdevel/storage-ng image
   # see https://github.com/yast/docker-storage-ng/blob/master/storage-ng-travis-ruby
-  - docker run -it -e TRAVIS=1 -e TRAVIS_JOB_ID="$TRAVIS_JOB_ID" yast-storage-ng-image storage-ng-travis-ruby
+  - docker run -it -e TRAVIS=1 -e TRAVIS_JOB_ID="$TRAVIS_JOB_ID" yast-storage-ng-image storage-ng-travis-ruby rpm -qa | sort
```

Comment:
Um, it should actually look like https://github.com/yast/yast-s390/blob/master/.travis.yml#L9

The `before_install` section is important as by default the `rpm` out will be collapsed by Travis (not bloating the log by default), but you can unwind it by clicking the line when needed.